tests/test_helper/microovn.bash

```diff
@@ -381,19 +381,19 @@ function microovn_wait_for_service_starttime() {
 # If cluster members do not reach "online" state before the MAX_RETRY is reached, this
 # function returns 1 as a return code.
 
-function wait_microovn_state(){
```

Comment:
unsure as to the reasoning for removing this generic function, realistically it could be useful, though I am biased on this.

Reply:
It wasn't anything personal :laughing: but there were some issues with `wait_microovn_{state,upgrading,online}`
* `wait_microovn_upgrading` was matching on `ONLINE` state as well, which means that it would return `true` when executed against fully online cluster
* `wait_microovn_online` and `wait_microovn_upgrading` were missing documentation
* `wait_microovn_state` documentation was outdated

And since the `wait_microovn_upgrading` became unused, I took the easy way out and removed it instead of fixing above issues.